### PR TITLE
[hotfix] Adding missing property to storage integration schema when using GCS driver

### PR DESCRIPTION
**Pull request by Sh1n**
## Test Plan

* [x] make test
* [x] provider used in a GCP live project

### CODE REVIEW

**Review by alldoami — CHANGES_REQUESTED** [review]
Hi! Thanks for the work @Sh1n ! Could you separate this into 2 PRs? One for the `storage_gcp_service_account` and the other for the notification integration?

**Review by alldoami — CHANGES_REQUESTED** [review]
Run `make docs` 😄

File: pkg/resources/notification_integration.go [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+			if err = d.Set("gcp_pubsub_subscription_name", v.(string)); err != nil {
+				return err
+			}
+		case "GCP_PUBSUB_SERVICE_ACCOUNT":
+			if err = d.Set("gcp_pubsub_service_account", v.(string)); err != nil {
+				return err
+			}
+
+		case "AZURE_CONSENT_URL":
+			if err = d.Set("azure_consent_url", v.(string)); err != nil {
```
Comment (alldoami) [inline pkg/resources/notification_integration.go:181]: you forgot to put `azure_consent_url` and `azure_multi_tenant_app_name` in the `notificationIntegrationSchema`

**Review by alldoami — CHANGES_REQUESTED** [review]
Rename PR to reflect your changes 😄

**Comment by alldoami** [comment]
@Sh1n please take a look at my comments.

**Comment by richardo-ps** [comment]
Anything we can do to help get this merged?

**Pull request closed without merging**